yew/src/virtual_dom/vtag.rs

```diff
@@ -145,7 +145,7 @@ impl TryFrom<VTag> for Html {
             for c in key.chars() {
                 let is_alnum = (c >= 'a' && c <= 'z') || (c >= '0' && c <= '9');
                 if !is_alnum && c != '-' && c != '_' && c != ':' {
-                    panic!("Invalid attribute name: `{}`", &key);
+                    return Err(HtmlStringifyError::InvalidAttributeName(key));
```

Comment:
I feel like `HtmlStringifyError` is a bit unwieldy – maybe it would be nicer to make it a bit shorter (something like `RenderError` or `HtmlRenderError`)?